src/redux/actions/login/login.test.js

```diff
@@ -1,5 +1,5 @@
 import userdata from '../../../__mocks__/loggedInUser';
-import { fetchError, fetchSucces } from '../../../__mocks__/window';
+import { fetchError, fetchSuccessfully, fetchClientError } from '../../../__mocks__/window';
```

Comment:
@habinezadalvan Did you remember to change these new functions names in other places as well

Reply:
I created another mocked function. I didn't use fetchSucces function.